website/config/routes.js

```diff
@@ -239,6 +231,7 @@ module.exports.routes = {
   'GET /docs/using-fleet/updating-fleet': '/docs/deploying/upgrading-fleet',
   'GET /blog':                   '/articles',
   'GET /brand':                  '/logos',
+  'GET /g':                      (req,res)=>{ return res.redirect(sails.config.custom.baseUrl+'?meet-fleet'+req.url.split('/g')[1]);},
```

Comment:
## 1
I think you may be missing an ampersand, no?

Also, while not essential, it's customary to include a `/` to represent the homepage, since sails.config.custom.baseUrl doesn't include one.

e.g. `fleetdm.com/?meet-fleet` rather than `fleetdm.com?meet-fleet`

## 2
> req.url.split('/g')[1])
Can you grab the query string out of [req.url](https://sailsjs.com/documentation/reference/request-req/req-url) a different way, for clarity?  For example, consider: `req.url.match(/\?(.+)$/) || req.url.match(/\?(.+)$/)[1]`

## 3
Also I think it's worth calling [`res.status(301);`](https://sailsjs.com/documentation/reference/response-res/res-status) to indicate a permanent redirect.

## 4
So you end up with something like:

```js
let originalQsWithoutQuestionMark = req.url.match(/\?(.+)$/) ? req.url.match(/\?(.+)$/)[1] : '';
res.status(301);// permanent redirect
return res.redirect(sails.config.custom.baseUrl+'/?meet-fleet&'+originalQsWithoutQuestionMark);
```

Look right?

## 5
Can you be sure and test that thoroughly to make sure it behaves properly and the URL bar looks right, both with navigating to:
- `/g`
- `/g?foo=bar&baz=bing`
- `/g?foo=bar`
- `/g?foo=bar&q=worlds%20largest%20dogs`  (make sure URL encoded characters don't get double-escaped)

Reply:
That makes sense, I've updated the redirect function in this PR to be 
```
(req,res)=> { let originalQueryString = req.url.match(/\?(.+)$/) ? '&'+req.url.match(/\?(.+)$/)[1] : ''; return res.redirect(301, sails.config.custom.baseUrl+'/?meet-fleet'+originalQueryString);},
```

All URLS tested give a 301 redirect with the original query string attached